internal/cmd/cmd.go

```diff
@@ -159,18 +159,18 @@ func extractHelp(command string, data []byte, longHelpTermRenderer, exampleTermR
 	state := stateReadTitle
 	var longHelpLines []string
 	var exampleLines []string
-	longFlags := make(map[string]bool)
-	shortFlags := make(map[string]bool)
+	longFlags := chezmoiset.New[string]()
+	shortFlags := chezmoiset.New[string]()
 
 	stateChange := func(line string, state *stateType) bool {
 		switch {
-		case strings.HasPrefix(line, "## Flags") || strings.HasPrefix(line, "## Common flags"):
+		case line == "## Flags" || line == "## Common flags":
 			*state = stateInOptions
 			return true
-		case strings.HasPrefix(line, "## Example"): // accept singular and plural
+		case line == "## Example":
```

Comment:
My idea was to support `## Example` or `## Note` section if someone makes a typo or intentionally uses singular form.
```suggestion
		case line == "## Examples":
```

Reply:
Good catch, thanks!